<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="1" total="2" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-10-18T17:01:26 IST" name="Default suite" finished-at="2022-10-18T17:01:36 IST" duration-ms="10483">
    <groups>
    </groups>
    <test started-at="2022-10-18T17:01:26 IST" name="Default test" finished-at="2022-10-18T17:01:36 IST" duration-ms="10483">
      <class name="facebook.test1">
        <test-method is-config="true" signature="all_objects()[pri:0, instance:facebook.test1@4f6ee6e4]" started-at="2022-10-18T17:01:26 IST" name="all_objects" finished-at="2022-10-18T17:01:28 IST" duration-ms="2281" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- all_objects -->
        <test-method is-config="true" signature="login_page()[pri:0, instance:facebook.test1@4f6ee6e4]" started-at="2022-10-18T17:01:28 IST" name="login_page" finished-at="2022-10-18T17:01:29 IST" duration-ms="1402" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login_page -->
        <test-method signature="validate_home_page()[pri:0, instance:facebook.test1@4f6ee6e4]" started-at="2022-10-18T17:01:29 IST" name="validate_home_page" finished-at="2022-10-18T17:01:33 IST" duration-ms="3468" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [https://www.facebook.com] but found [https://www.facebook.com/]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [https://www.facebook.com] but found [https://www.facebook.com/]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at facebook.test1.validate_home_page(test1.java:47)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validate_home_page -->
        <test-method is-config="true" signature="close_browser()[pri:0, instance:facebook.test1@4f6ee6e4]" started-at="2022-10-18T17:01:33 IST" name="close_browser" finished-at="2022-10-18T17:01:33 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close_browser -->
        <test-method is-config="true" signature="logout_(org.testng.ITestResult)[pri:0, instance:facebook.test1@4f6ee6e4]" started-at="2022-10-18T17:01:33 IST" name="logout_" finished-at="2022-10-18T17:01:36 IST" duration-ms="3287" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=validate_home_page status=FAILURE method=test1.validate_home_page()[pri:0, instance:facebook.test1@4f6ee6e4] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout_ -->
      </class> <!-- facebook.test1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
